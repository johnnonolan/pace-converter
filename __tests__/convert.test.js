const convert = require('../src/convert');

describe('convert', () => {
  const scenarios = [
    {
      pace: '4:30',
      unit: 'minkm'
    },
    {
      pace: '4:40',
      unit: 'minkm'
    },
    {
      pace: '5:00',
      unit: 'minkm'
    }
  ];

  scenarios.forEach((scenario) => {
    test(`when ${scenario.pace} ${scenario.unit} expect model to match snapshot`, () => {
      const model = convert(scenario.pace, scenario.unit);
      expect(model).toMatchSnapshot();
    });
  });
});
