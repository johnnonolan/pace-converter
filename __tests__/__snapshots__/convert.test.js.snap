// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`convert when 4:30 minkm expect model to match snapshot 1`] = `
Object {
  "kmh": 13.333,
  "mih": 8.285,
  "minkm": "4:30",
  "minmi": "7:14.52",
}
`;
